### PR TITLE
Feat/user resources

### PR DESCRIPTION
**Pull request by Cliftonz**
Made requested changes. Please see make any changes necessary to make this up to standards or let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+This is the web site is for the CSG web page.
```
Comment (TheOtherUnknown) [inline README.md]: Typo

File: style.css [hunk 2]
```diff
@@ -74,6 +74,23 @@ body{
 .officers-body {
     padding: 30px 0px 30px 0px;
 }
+
+.calendar-container {
+    position: relative;
+    padding-bottom: 75%;
```
Comment (TheOtherUnknown) [inline style.css]: This causes problems on mobile AND desktop. 
Screenshot

File: style.css [hunk 3]
```diff
@@ -74,6 +74,23 @@ body{
...
+    height: 0;
+    overflow: hidden;
+}
+
+.form-container {
+    background-color: #FDEEE8;
+    position: relative;
+    padding-bottom: 75%;
```
Comment (TheOtherUnknown) [inline style.css]: Also causes problems on both.

File: index2.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (TheOtherUnknown) [inline index2.html]: There's a much easier, Jekyll way to do this. Here's an example: https://github.com/jekyll/example/blob/gh-pages/about.md

File: _layouts/CSGLabs.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (TheOtherUnknown) [inline _layouts/CSGLabs.html]: There's a much easier, Jekyll way to do this. Here's an example: https://github.com/jekyll/example/blob/gh-pages/about.md

File: _includes/CSGLabsContributers.html [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<div class="container CSGLabs-temp">
```
Comment (TheOtherUnknown) [inline _includes/CSGLabsContributers.html]: There's a much easier, Jekyll way to do this. Here's an example: https://github.com/jekyll/example/blob/gh-pages/about.md

File: index2.html (hunk 4 above)
Comment (Cliftonz) [reply index2.html]: I can not get that way to work. Will convene with you in the future and fix later.

File: style.css (hunk 2 above)
Comment (Cliftonz) [reply style.css]: What would be the best way to fix this ?

Comment (Cliftonz) [reply style.css]: @TheOtherUnknown

Comment (TheOtherUnknown) [reply style.css]: Letting Bootstrap take care of it would be ideal. We may need to modify the iframe itself.

**Pull request merged**